packages/genymotion-bin/genymotion-bin.pacscript

```diff
@@ -5,14 +5,45 @@ pkgver="3.6.0"
 replace=("${gives}")
 breaks=("${gives}" "${gives}-bin" "${gives}-git" "${gives}-app" "${gives}-deb")
 arch=("amd64")
-depends=("xdg-utils")
 hash="092d40f6e76de3b6e1827609aaa9021b7cf85da3d51e6cf8d7b553b18d9c70e0"
 url="https://dl.genymotion.com/releases/genymotion-${pkgver}/genymotion-${pkgver}-linux_x64.bin"
 homepage="https://genymotion.com"
 pkgdesc="Android virtual devices"
 maintainer="Arrowsome <ramtintoosi@gmail.com>"
 
+_GENYMOTION_FILE="./genymotion-${pkgver}-linux_x64.bin"
+_GENYMOTION_INSTALLDIR="${pkgdir}/opt/genymotion"
+
+prepare() {
+  sudo rm -rf "/opt/${gives}"
+  sudo mkdir -p "${_GENYMOTION_INSTALLDIR}"
+
+  sudo chmod +x "${_GENYMOTION_FILE}"
```

Comment:
Where is this being installed in/before `prepare`?

Reply:
hmm, what do you mean by "this"? I don't understand the question!

Reply:
Oh sorry I was tripping yesterday. I don't know why I didn't read the entire script.